alacritty/Cargo.toml

```diff
@@ -55,12 +55,11 @@ xdg = "2.5.0"
 png = { version = "0.17.5", default-features = false, optional = true }
 
 [target.'cfg(target_os = "macos")'.dependencies]
-cocoa = "0.24.0"
+cocoa = "0.25.0"
 objc = "0.2.2"
 
 [target.'cfg(not(any(target_os="windows", target_os="macos")))'.dependencies]
 x11-dl = { version = "2", optional = true }
-wayland-client = { version = "0.29.0", features = ["dlopen"], optional = true }
```

Comment:
@kchibisov was this added to force the `dlopen` feature, or did we actually use it in the past? It seems like that got removed from the latest wayland-client versions.

If the feature name was just changed and we want this back, let me know.

Reply:
you can remove it, it's fine.